src/parquet/util/rle-encoding.h

```diff
@@ -171,8 +171,12 @@ class RleEncoder {
 
   /// Returns the maximum byte size it could take to encode 'num_values'.
   static int MaxBufferSize(int bit_width, int num_values) {
-    int bytes_per_run = BitUtil::Ceil(bit_width * MAX_VALUES_PER_LITERAL_RUN, 8.0);
```

Comment:
Should we add another variable `MIN_VALUES_PER_LITERAL_RUN` to 8 and use the same previous logic here ?